src/Docfx.DataContracts.Common/Constants.cs

```diff
@@ -108,4 +108,15 @@ public static class TableOfContents
         public const string MarkdownTocFileName = "toc.md";
         public const string YamlTocFileName = "toc.yml";
     }
+
+    public static class EnvironmentVariables
```

Comment:
nit: is it easier if we also read the environment variables here, similar to https://github.com/dotnet/docfx/blob/v3/src/docfx/config/EnvironmentVariable.cs#L6?

Reply:
ok. I try to rewrite code with suggested styles.
(And refactor other codes that use environment variables)

Reply:
I've added suggested style helper methods/properties.
And refactor other codes that use environment variable

`GitUtility.cs` file remains as is. 
Because `Docfx.Common` project don't have reference to `Docfx.DataContracts.Common` project.